syntax = "proto3";

import "hyro/msgs/common/Header.proto";
import "hyro/msgs/navigation/Pose.proto";

package hyro.msgs.ar_object_localization;

message ObjectLocalizationItem
{
  string object_name=1;
  hyro.msgs.navigation.Pose pose = 2;
  float confidence=3;
}

message ObjectLocalization
{
  hyro.msgs.common.Header header = 1;
  repeated ObjectLocalizationItem localizations=2;
}
